docs/using/config/index.rst

```diff
@@ -1,6 +1,6 @@
-===================
-Configuring Flocker
-===================
+==================
+Setting up Flocker
```

Comment:
This isn't super accurate either. "Setting up Flocker" sounds just like "Install Flocker". This isn't about setting *flocker* up, it's about setting up datasets and applications *using* Flocker.